src/Groups/Groups/examples/v1.0-beta/Get-MgGroupLifecyclePolicy.md

```diff
@@ -1,14 +1,15 @@
-### Example 1: Using the Get-MgGroupLifecyclePolicy Cmdlet
-```powershell
-Import-Module Microsoft.Graph.Groups
-Get-MgGroupLifecyclePolicy -GroupLifecyclePolicyId $groupLifecyclePolicyId
-```
-This example shows how to use the Get-MgGroupLifecyclePolicy Cmdlet.
-To learn about permissions for this resource, see the [permissions reference](/graph/permissions-reference).
-### Example 2: Using the Get-MgGroupLifecyclePolicy Cmdlet
-```powershell
-Import-Module Microsoft.Graph.Groups
-Get-MgGroupLifecyclePolicy
-```
-This example shows how to use the Get-MgGroupLifecyclePolicy Cmdlet.
-To learn about permissions for this resource, see the [permissions reference](/graph/permissions-reference).
+### Example 1: Code snippet
+
+```powershell
Import-Module Microsoft.Graph.Groups
+
+$params = @{
+	groupLifetimeInDays = 180
+	managedGroupTypes = "Selected"
+	alternateNotificationEmails = "admin@contoso.com"
+}
+
+Update-MgGroupLifecyclePolicy -GroupLifecyclePolicyId $groupLifecyclePolicyId -BodyParameter $params

```

Comment:
Cmdlet mismatch - `Get-MgGroupLifecyclePolicy` vs `Update-MgGroupLifecyclePolicy`. @timayabi2020, please review the matching logic on example update script. This is a good example to start with.